src/UI/Seller/src/app/products/components/product-edit/product-edit.component.html

```diff
@@ -1415,7 +1400,9 @@ <h5 translate>ADMIN.PRODUCT_EDIT.DOCUMENTS_PRODUCTS</h5>
   </form>
   <!-- TODO: 2/2 Abstract into component to be used anywhere using inputs and outputs -->
   <div
-    *ngIf="userContext?.UserType === 'SUPPLIER'"
+    *ngIf="
+      userContext?.UserType === 'SUPPLIER' || userContext?.UserType === 'SELLER'
+    "
```

Comment:
Is this ngIf necessary? The only user types allowed in the admin application are either seller or supplier

Reply:
Agreed.  I removed it.